app/src/main/java/com/pajato/android/gamechat/common/InvitationManager.java

```diff
@@ -303,40 +306,45 @@ public void extendAppInvitation(final FragmentActivity fragmentActivity, final S
 
     public void onResult(@NonNull AppInviteInvitationResult result) {
         Log.i(TAG, "getInvitation intent=" + result.getInvitationIntent());
-        if (result.getStatus().isSuccess()) {
-            // Extract deep link from Intent
-            Intent intent = result.getInvitationIntent();
-            String deepLink = AppInviteReferral.getDeepLink(intent);
-            Log.i(TAG, "getInvitation with deepLink: " + deepLink);
-            // If we have a deep link, try to find the groupKey. Also, extract the commonRoomKey
-            // and groupName values.
-            if (deepLink != null && !deepLink.equals("")) {
-                Uri dlUri = Uri.parse(deepLink);
-                String firebaseLink = dlUri.getQueryParameter("link");
-                String commonRoomKey = dlUri.getQueryParameter("commonRoomKey");
-                String groupName = dlUri.getQueryParameter("groupName");
-
-                if (firebaseLink != null && !firebaseLink.equals("")) {
-                    Uri fbUri = Uri.parse(firebaseLink);
-                    // Get the group key if one was specified
-                    List<String> parts = fbUri.getPathSegments();
-                    if (parts.contains("groups")) {
-                        String groupKey = parts.get(parts.lastIndexOf("groups") + 1); // the group key is after 'groups'
-                        Log.i(TAG, "getInvitation: groupKey=" + groupKey);
-                        mInvitedGroups.put(groupKey, new GroupInviteData(groupName, commonRoomKey));
-                    } else {
-                        Log.i(TAG, "getInvitation: no groupKey specified");
-                    }
-                } else {
-                    Log.e(TAG, "getInvitation: can't get group key - firebaseLink is not set");
-                }
-            } else {
-                Log.e(TAG, "getInvitation: can't get group key - deepLink is not set");
-            }
-
-        } else {
+        if (!result.getStatus().isSuccess()) {
             Log.i(TAG, "getInvitation: no deep link found.");
+            return;
         }
+
+        // Extract deep link from Intent
+        Intent intent = result.getInvitationIntent();
+        String deepLink = AppInviteReferral.getDeepLink(intent);
+        Log.i(TAG, "getInvitation with deepLink: " + deepLink);
+        // If deep link doesn't exist, just log and return
```

Comment:
fwiw, I'd be inclined to combine this comment with the previous one by moving it up and smashing the two together.